01week/rockPaperScissors.js

```diff
@@ -8,9 +8,68 @@ const rl = readline.createInterface({
 });
 
 
+
 function rockPaperScissors(hand1, hand2) {
 
-  // Write code here
+  // create a sanitized array of the two hands for easier manipulation
+  let handArr = [hand1, hand2].map((x) => x.toLowerCase().replace(/\s/g, ''));
+
+  // declare an array of accepted hand values
+  const validHands = ['rock', 'paper', 'scissors', 'lizard', 'spock'];
+  
+  // return an error if the hands chosen are not in the array of accepted values
+  for (let value of handArr) {
```

Comment:
If you got rid of the array, you wouldn't need this loop. I get that loops are nice, but the more that you can cut them out, the faster your code will run.